CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+* Fix for duplicate dotted keys ([#46])
```

Comment:
This changelog message is a little opaque, maybe "Fixed `StructlogFormatter` to use the `message` field instead of `event` for the log message. This fixes an issue where field conflicts would occur when used with the `elasticapm` package.

Also the PR URL doesn't look right yet

Reply:
Thanks, this was from before I had this fixed properly. I'll get it fixed.